### PR TITLE
20.6.0 Release Notes

### PR DESCRIPTION
**Pull request by jochman**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.6.0.md [hunk 1]
```diff
@@ -0,0 +1,865 @@
...
+##### __GetTime__  
+Added support for subtracting days and years from current date using **daysAgo** and **yearsAgo** arguments.  
+
+---
+
+### Common Types Pack v1.0.1
+
+#### Classifiers
+##### __Mail Listener - Classifier__  
+Updated classifier according to new schema.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:191]: lets remove all `Updated classifier according to new schema.`, its for 6.0 and above

File: content-repo/extra-docs/releases/20.6.0.md [hunk 2]
```diff
@@ -0,0 +1,865 @@
+# Cortex XSOAR Content Release Notes for version 20.6.0 (56612)
+##### Published on 09 June 2020
+
+**End Of Life Notice**: The following Integrations were deprecated on Nov 2019:
+- **Azure Compute**
+- **Azure Security Center**
+
+These Integrations will reach the end of life on July 31, 2020, due to changes to the back-end authentication services needed for these Integrations. Use the Azure Compute v2 and Azure Security Center v2 Integrations instead.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:8]:
```suggestion
These Integrations will reach the end of life on July 31, 2020, due to changes to the back-end authentication services needed for these integrations. Use the Azure Compute v2 and Azure Security Center v2 Integrations instead.
```

File: content-repo/extra-docs/releases/20.6.0.md [hunk 3]
```diff
@@ -0,0 +1,865 @@
...
+##### __Active Directory Query v2__  
+Improved parsing of custom attributes in the ***ad-get-user*** command.
+
+---
+
+### ArcSight ESM Pack v1.0.1
+
+#### Integrations
+##### __ArcSight ESM v2__  
+Fixed a breaking change in the outputs of the ***as-get-security-events*** command.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:57]: this is from previous release, can remove

File: content-repo/extra-docs/releases/20.6.0.md [hunk 4]
```diff
@@ -0,0 +1,865 @@
...
+#### Integrations
+##### __Check Point__  
+Added the **command** argument to the ***checkpoint*** command. This argument is the command to run in the firewall.
+
+---
+
+### Chronicle Pack v1.0.1
+#### Integrations
+##### __Google Chronicle Backstory__  
+Integration name has been updated
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:132]: we can remove that one

File: content-repo/extra-docs/releases/20.6.0.md [hunk 5]
```diff
@@ -0,0 +1,865 @@
...
+  - Added support for filtering incident creation by Expanse Exposure severity level.
+
+
+---
+
+### FalconHost Pack v1.1.0
+
+#### Integrations
+##### __FalconHost__  
+Added support for 3 commands from the Threat graph API:
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:271]:
```suggestion
Added support for 3 commands from the Threat Graph API:
```

File: content-repo/extra-docs/releases/20.6.0.md [hunk 6]
```diff
@@ -0,0 +1,865 @@
...
+Fixed image paths in documentation.
+
+
+---
+
+### HelloWorld Pack v1.1.2
+
+#### Scripts
+##### __HelloWorldScript__  
+  - Grammar correction for code description.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:347]: can remove that one

File: content-repo/extra-docs/releases/20.6.0.md [hunk 7]
```diff
@@ -0,0 +1,865 @@
...
+
+##### __Illusive Networks - Incoming Mapper__  
+Added incoming mapper.
+---
+
+### Indeni Pack v1.0.1
+
+#### Playbooks
+##### __Indeni Demo__  
+---
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:401]: remove

File: content-repo/extra-docs/releases/20.6.0.md [hunk 8]
```diff
@@ -0,0 +1,865 @@
...
+##### __integration-SafeBreach_README.md__  
+Fixed image paths in documentation.
+
+---
+
+### Salesforce Pack v1.0.1
+
+#### Integrations
+##### __integration-Salesforce_README.md__  
+Fixed image paths in documentation.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:679]: remove

File: content-repo/extra-docs/releases/20.6.0.md [hunk 9]
```diff
@@ -0,0 +1,865 @@
...
+  - Fixed an issue in which acknowledged detections were fetched as incidents.
+
+
+---
+
+### SafeBreach - Breach and Attack Simulation platform Pack v1.0.1
+
+#### Integrations
+##### __integration-SafeBreach_README.md__  
+Fixed image paths in documentation.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:671]: remove

File: content-repo/extra-docs/releases/20.6.0.md [hunk 10]
```diff
@@ -0,0 +1,865 @@
...
+##### __Smokescreen IllusionBLACK - Classifier__  
+Added default classifier and mapper.
+
+---
+
+### SplunkPy Pack v1.0.2
+
+#### Integrations
+##### __SplunkPy__  
+- Added support for HTTPS handler, which uses the Python **requests** library.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:765]: this was released already

File: content-repo/extra-docs/releases/20.6.0.md [hunk 11]
```diff
@@ -0,0 +1,865 @@
...
+
+#### Integrations
+##### __README.md__  
+Fixed image paths in documentation.
+---
+
+### US - Breach Notification Pack v1.0.2
+
+#### IncidentTypes
+##### __US Breach Notification__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.0.md:834]: is this a new one?

File: content-repo/extra-docs/releases/20.6.0.md [hunk 12]
```diff
@@ -0,0 +1,790 @@
...
+###### Humio QueryJob Poll
+Run and poll a Humio Query Job.
+
+---
+
+### AWS Feed Pack v1.0.1
+
+#### Classifiers
+##### __AWS Feed - Classifier__  
+Updated classifier according to new schema.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:41]: I think we can remove unless there is an update in the logic.

File: content-repo/extra-docs/releases/20.6.0.md [hunk 13]
```diff
@@ -0,0 +1,790 @@
...
+#### Integrations
+##### __Carbon Black Defense__  
+ - Fixed a bug in the ***cbd-find-events*** command where hash arguments were not processed correctly.
+
+---
+
+### Carbon Black Enterprise Live Response Pack v1.0.1
+
+#### Integrations
+##### __carbonblackliveresponse__
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:99]: Change to display name

File: content-repo/extra-docs/releases/20.6.0.md [hunk 14]
```diff
@@ -0,0 +1,790 @@
...
+#### Integrations
+##### __carbonblackliveresponse__  
+Fixed an error when pulling a file in the ***get-file-from-endpoint*** command.
+
+---
+
+### Carbon Black Enterprise Response Pack v1.0.3
+
+#### Integrations
+##### __carbonblack-v2__
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:107]: Use display name

File: content-repo/extra-docs/releases/20.6.0.md [hunk 15]
```diff
@@ -0,0 +1,790 @@
...
+##### __Phishing Playbook - Manual__  
+Added the **Phishing Playbook - Manual** playbook to the DeprecatedContent pack.
+
+---
+
+### Deprecated Content Pack v1.0.4
+
+#### Integrations
+##### __SafeBreach__  
+Use the **SafeBreach v2** integration instead.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:237]:
```suggestion
Deprecated. Use the **SafeBreach v2** integration instead.
```

File: content-repo/extra-docs/releases/20.6.0.md [hunk 16]
```diff
@@ -0,0 +1,548 @@
...
+### Microsoft Graph Security Pack v2.0.0
+
+#### Integrations
+##### __Microsoft Graph__  
+- Added fetch-incidents function and unit test
+
+
+---
+
+### Microsoft Intune Feed Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:300]: I think you can remove this pack

**Comment by glicht** [comment]
@jochman see my last commit

File: content-repo/extra-docs/releases/20.6.0.md [hunk 17]
```diff
@@ -0,0 +1,548 @@
...
+
+### Demisto REST API Pack v1.0.1
+
+#### Integrations
+##### __Demisto REST API__  
+Fixed an issue in which URL endpoints given with forward slash character as prefix were not processed as expected. 
+
+---
+
+### Deprecated Content Pack v1.0.4
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:182]: Let's remove this pack.

File: content-repo/extra-docs/releases/20.6.0.md [hunk 18]
```diff
@@ -458,11 +444,11 @@ Added verification to the **Corporation Name** parameter to match the required p
 
 #### Integrations
 ##### __SlackV2__  
-Update default integration parameter to new brand name Palo Alto Networks XSOAR (formerly Demisto).
+Updated default integration parameter to new brand name Palo Alto Networks XSOAR (formerly Demisto).
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.0.md:447]:
```suggestion
Updated default integration parameter to new brand name Palo Alto Networks Cortex XSOAR (formerly Demisto).
```

**Comment by glicht** [comment]
@jochman added a commit

**Comment by jochman** [comment]
> Looks good. See my 2 last commits.

in the next release the rn collector will use the displayName instead of name.

**Pull request merged**